sanitize_html/lib/src/html_formatter.dart

```diff
@@ -0,0 +1,67 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import 'dart:convert';
+
+import 'package:html/dom.dart';
+
+final _attrEscape = HtmlEscape(HtmlEscapeMode.attribute);
+final _textEscape = HtmlEscape(HtmlEscapeMode.unknown);
```

Comment:
shouldn't this be: https://api.dartlang.org/stable/2.3.1/dart-convert/HtmlEscapeMode/element-constant.html